src/cta-button/editor.scss

```diff
@@ -0,0 +1,97 @@
+/**
+ * #.# Editor Styles
+ *
+ * CSS for just Backend enqueued after style.scss
+ * which makes it higher in priority.
+ */
+
+.bulma-blocks-editor-link-card {
+
+  padding: 0.875rem;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #000;
```

Comment:
perhaps this could be changed to ```#000000``` to remain consistent with the rest of the color hexcodes